### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by NCC1701M** [mtxr/vscode-sqltools#831] [Q&A] [answered]
Hello @mtxr ,

there are some issues open but it seems that nothing happens. Is this project still alive?

**Comment by DannyRodrigues** [comment]
Looks like there's some new maintainers now as mentioned in an issue from May 2022: #896 

I haven't noticed a lot of activity since then though.

**Comment by gjsjohnmurray** [comment, marked as answer]
It's taken us a while to get up to speed, but see #941 for the first release we've achieved.
